Tests/ParseSwiftTests/ParseFacebookCombineTests.swift

```diff
@@ -84,7 +84,22 @@ class ParseFacebookCombineTests: XCTestCase { // swiftlint:disable:this type_bod
         #endif
         try ParseStorage.shared.deleteAll()
     }
+    func testMakeDictionary() {
```

Comment:
Can you move this test to ParseFacebookCombineTests

Reply:
I ended up removing this because it's already covered by test (just the keys are wrong)